cmd/webhook/main.go

```diff
@@ -126,6 +132,9 @@ func NewValidationAdmissionController(ctx context.Context, cmw configmap.Watcher
 
 		// Whether to disallow unknown fields.
 		true,
+
+		// Extra validating callbacks to be applied to resources.
+		callbacks,
```

Comment:
It'll be off by default. I have it so that it will only attempt the extra dryrun validation if we are attempting a dryrun ourselves. I'll use that for the e2e test and we can let it bake and later knock it out.